.vscode/launch.json

```diff
@@ -70,45 +48,5 @@
                 "${workspaceRoot}"
             ]
         },
-        {
-            "name": "RSpec - active spec file only",
```

Comment:
I was just cleaning up until we add more, trying to make the debugger options cleaner. I just left all the ones we mentioned in the README and figured if we needed more we'd just add them as they were usable. I don't have a strong opinion one way or another on this, I can easily add it back.